e2e/vertical-full-page-map.spec.ts

```diff
@@ -1,15 +1,31 @@
 import { test, expect } from '@playwright/test';
 
+async function getAnswersVerticalResults(page) {
```

Comment:
nit: maybe this could be renamed to better explain it's purpose, e.g. `waitForResultsLoad` or `waitForResultsToRender`